.github/workflows/build-and-test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     - name: Setup .NET
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: 3.1.x
+        dotnet-version: 6.0.x
```

Comment:
Will the netcoreapp3.1 have testing as well, both through actions and CI? It probably should if you'll continue to support it by updating dependencies and bug fixes.

Reply:
We should set that up specifically for the v3.1 branch. The v6.0 Entity Framework provider will not have support for .NET 3.1.